matchms/filtering/__init__.py

```diff
@@ -1,3 +1,23 @@
+"""Functions for processing mass spectra.
+Provided functions will usually only perform a single action to a spectrum.
+This can be changes or corrections of metadata, or peak filtering.
+More complicated processing pipelines can be build by stacking several of
+the provided filters.
+
+Example of how to use a single filter function:
+
+.. code-block:: python
+
+    import numpy as np
+    from matchms import Spectrum
+    from matchms.filtering import filter_of_choice
+
+    spectrum = Spectrum(mz=np.array([100, 120, 150, 200.]),
+                          intensities=np.array([0.7, 1.0, 0.2, 0.001]),
+                          metadata={'id': 'spectrum1'})
+    spectrum_filtered = filter_of_choice(spectrum)
```

Comment:
Dont use hypothetical filter 
```suggestion
    from matchms.filtering import make_charge_scalar

    spectrum = Spectrum(mz=np.array([100, 120, 150, 200.]),
                          intensities=np.array([0.7, 1.0, 0.2, 0.001]),
                          metadata={'id': 'spectrum1', 'charge': ['+1']})
    spectrum_filtered = make_charge_scalar(spectrum)
```

Reply:
With real filter method its also possible to rewrite to doctest

Reply:
I now added a tested code example.

Reply:
Nice, test passed on CI

```
Document: api/matchms.filtering
-------------------------------
1 items passed all tests:
   1 tests in default
1 tests in 1 items.
1 passed and 0 failed.
Test passed.
```